### DISCUSSION TITLE
Hide file fields in drawer-item

### DISCUSSION THREAD

**Discussion by martijnboland** [directus/directus#3620] [Archive]
When editing file (image) properties in a drawer, all fields are editable, including the ones that are normally displayed in the sidebar. Our users have no idea what to do with these fields. Is there a possibility to also hide these fields in when editing a file in a drawer?

![Screenshot 2021-01-08 at 15 35 48](https://user-images.githubusercontent.com/167455/104027609-e955fb00-51c7-11eb-9b0d-ec5cfd67af60.png)

**Comment by benhaynes** [comment]
Thanks @martijnboland — I'd say this is actually a bug. GitHub recently removed the "Transfer Discussion to Issue"... do you think you could recreate this as an issue so we can triage it appropriately?

**Reply by rijkvanzanten** [reply]
Not a bug.. Those fields are normally shown in the sidebar when looking at the file detail page, but when accessing a file through a relationship in the drawer, they're appended at the end of the form like regular fields. The only problem here is that they're supposed to be readonly

**Reply by martijnboland** [reply]
Read only makes sense to me

**Reply by rijkvanzanten** [reply]
@martijnboland Would you like to open an issue for that, or shall I?

**Reply by benhaynes** [reply]
Since we have a custom detail page for Files, it might be worth discussing how we could catch/override the Files/Users drawers so that they are bit more polished (eg: those fields that are normally in the sidebar). Should be adding more field metadata for them, right?

**Reply by rijkvanzanten** [reply]
> Should be adding more field metadata for them, right?

Yeah! We should update them in the api so they're readonly, have some icons / placeholders, use the right interfaces, monospace etc etc

**Reply by martijnboland** [reply]
@rijkvanzanten issue created: https://github.com/directus/directus/issues/3623
